src/styles/themer/withTheme.js

```diff
@@ -4,6 +4,10 @@ import React, { Component } from 'react'
 import themer from './index'
 import { themePropTypes } from './utils'
 
+const isStateless = (Component) => {
+  return Component.prototype && !Component.prototype.render
```

Comment:
I don't think we need to check the presence of `prototype`

Reply:
True